.golangci.yaml

```diff
@@ -0,0 +1,73 @@
+########
+# NOTE
+#
+#  This file is duplicated in the following repos:
+#    - operator-framework/kubectl-operator
+#    - operator-framework/catalogd
+#    - operator-framework/operator-controller
+#    - operator-framework/operator-registry
+#
+#  If you are making a change, please make it in ALL
+#  of the above repositories!
+#
+#  TODO: Find a way to have a shared golangci config.
+########
+
+run:
+  # Default timeout is 1m, up to give more room
+  timeout: 4m
+
+linters:
```

Comment:
you don't have to, but if it's easy enough, we could add govet here as well (then maybe drop it from the Makefile if it's there)

Reply:
the .golangci.yaml file in this project is a copy of that in op-con.  Since that one has just updated, I can take that change, but IMO we can update the Makefile in a separate PR.  This one is already way beyond any reasonable complexity.

Reply:
Hi @grokspawn 

You are with an outdated version :-P 
See that we added govet there and we cleanup the staff: https://github.com/operator-framework/operator-controller/commit/099a6cf8ce5f38f35a109f27b49f8ae2f0129290

Reply:
But I am ok with we move forward as it is and add govet in a follow up. 
It is fine for me too. @perdasilva wdyt?

Reply:
I did add govet in [ac872deec7a3f69452cdf7d1cdcc38b7dec70911](https://github.com/operator-framework/operator-registry/pull/1524/commits/ac872deec7a3f69452cdf7d1cdcc38b7dec70911)